page-components/expandables.md

```diff
@@ -28,25 +28,28 @@ Expandables are used for displaying non-essential information on a page.
       <p>Expandables in the Capital Framework</p>
     </li>
   </ul>
-</div> 
+</div>
 
 <h2 id="use">Use<span class="cf-code-link"><a href="https://cfpb.github.io/cf-expandables/docs/">View code <span class="cf-icon cf-icon-external-link"></span></a></span></h2>
 
 The `find` function will not discover information hidden by a collapsed expandable, so use good judgement in deciding which information to hide.
+Non-javascript users should default to the expanded state. Otherwise, the default state (expanded or collapsed) will depend on the circumstance.
 
-Non-javascript users should default to the expanded state. Otherwise, the default state (expanded or collapsed) will depend on the circumstance.  
+<h2 id="style">Style</h2>
 
-<h2 id="style">Style<span class="cf-code-link"><a href="https://cfpb.github.io/cf-expandables/docs/">View code <span class="cf-icon cf-icon-external-link"></span></a></span></h2>
+<h3 class="h4">Individual</h3>
 
 <div class="content-33 content-first">
 
-Standard expandables feature a colored bar, a header, the expand/collapse minicon within a circle and a label (if space allows). Ideally, the entire bar is actionable. The header should clearly indicate what the user will see when the content is expanded. It should not be a call to action.
+When only one expandable is used it should include a stroke on all sides, heading text, the expand/collapse minicon within a circle with a label reading “Show” or “Hide,” (when possible). A white or 5% grey background can be used to highlight the section from the page background.
```

Comment:
Insert comma after "one expandable is used". Insert "and " before "the expand/collapse minicon". Delete comma from '“Hide,”'. "Gray".